source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -95,6 +96,18 @@ class BioXASMainBeamline : public BioXASBeamline
 	/// Returns the I2 scaler channel detector.
 	virtual CLSBasicScalerChannelDetector* i2Detector() const { return i2Detector_; }
 
+	/// Returns the inboard 32Ge detector.
+	virtual BioXAS32ElementGeDetector* ge32ElementDetector() const { return ge32DetectorInboard_; }
```

Comment:
Wasn't there an issue where we had lists of Ge detectors in BioXASBeamline?  I forget the issue number.  Will this need to be modified to conform to that upon merge?